kindlecomicconverter/utils.py

```diff
@@ -0,0 +1,10 @@
+import os
+import subprocess
+
+def run_subprocess_silent(command, **kwargs):
```

Comment:
I would call this `subprocess_run_silent` to make it sounds similar to what it's wrapping.
I would also put it in the existing `shared.py`